addons/mail/static/src/components/thread_view/thread_view_tests.js

```diff
@@ -626,94 +702,6 @@ QUnit.test('new messages separator on posting message', async function (assert)
     );
 });
 
-QUnit.test('new messages separator on receiving new message', async function (assert) {
```

Comment:
what are reasons to delete this test?

Reply:
It was there 2 times on the file, the other is line 547

Reply:
ok